src/components/Solution.js

```diff
@@ -0,0 +1,26 @@
+import React, {Component} from 'react'
+import Letter from './Letter'
+
+class Solution extends Component{
+    generateLetterWord(word){
+        const lettersArray = []
+        for(let i in word){
+            if(this.props.letterStatus[word[i]]){
+                lettersArray.push(<Letter letter={word[i]} key={i} />)
+            }else{
+                lettersArray.push(<Letter letter={'_'} key={i} />)
+            }
+        }
```

Comment:
again, you could use a ternary instead of the if-else:
`lettersArray.push(<Letter letter={this.props.letterStatus[word[i]]? word[i] : '_'} key={i} />)`